### PR TITLE
Docs: improve local setup instructions

### PR DESCRIPTION
**Pull request by EL-HARBYLY-Abderrahman**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Some more installation steps for Widnows OS

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
Non

* **Other information**:

### CODE REVIEW

**Review by MeStrak — CHANGES_REQUESTED** [review]

File: docs/installation.md [hunk 1]
```diff
@@ -185,6 +185,21 @@ AWS_S3_ENDPOINT = http://localhost:4572/
...
 ```
 
+Some more steps :(All platforms)
+
+```bash
+# Setting node version
+nvm install 12.13.0
+
+# Check the node version in local, must be 12.13.0
+nvm version
```
Comment (MeStrak) [inline docs/installation.md:195]: This command isn't used to check the node version (it's only the node -v part which is required)

File: docs/installation.md [hunk 2]
```diff
@@ -185,6 +185,21 @@ AWS_S3_ENDPOINT = http://localhost:4572/
...
+node -v
+
+# AWS access keys configuration in the local. Setting dummy value for both so you can run tests
+set AWS_ACCESS_KEY_ID=dummy
```
Comment (MeStrak) [inline docs/installation.md:199]: This is Windows specific and should not be in the all environments section

File: docs/installation.md (hunk 1 above)
Comment (EL-HARBYLY-Abderrahman) [reply docs/installation.md]: Done

File: docs/installation.md (hunk 2 above)
Comment (EL-HARBYLY-Abderrahman) [reply docs/installation.md:199]: Done, i make a warning section. I hope will working inside a bash

**Pull request merged**